app/assets/stylesheets/_returns.scss

```diff
@@ -11,60 +11,3 @@ th.govuk-table__header.row-border {
 td.govuk-table__cell.row-border {
   border-bottom: 1px solid #b1b4b6 !important;
 }
-
-.hmrc-user-research-banner {
```

Comment:
This apply the same fixes of Pan to the search banner

Reply:
Do we need this file at all? 
It feels weird to have returns.css sheet in reg...

Reply:
I am nervous to remove the css completely as I am not sure if these are used by imported components. What I can do to be safe is to move what is in _returns.scss to _registration.scss and remove the _return.scss file, as I see your point Pan.

Reply:
Yeah sounds like a better idea than leaving this one here, custom CSS is to be avoided as much as possible tbh

Reply:
done